src/PKSim.Core/Model/ExpressionProfile.cs

```diff
@@ -0,0 +1,11 @@
+﻿using OSPSuite.Core.Domain;
+
+namespace PKSim.Core.Model
+{
+   public class ExpressionProfile : PKSimBuildingBlock
+   {
+      public ExpressionProfile():base(PKSimBuildingBlockType.ExpressionProfile)
```

Comment:
@Yuri05 This is the default building block implementation for now. Behavior to come next :)